linux_service/start_moneo_services.sh

```diff
@@ -1,6 +1,47 @@
 #!/bin/bash
-PUBLISHER=$1
-MONEO_PATH=$2
+PUBLISHER=$3
```

Comment:
This name seems a bit confusing for this variable if it is supposed to be true or false? Could it be called WITH_PUBLISHER or something?

Reply:
ok I addressed. Please have another look.